### PR TITLE
Map

### PR DESCRIPTION
**Pull request by akramerin**
Added "address" to the model. 
Added "address" seeds.
Added mapbox.
Added "address" to the "New Book" Form (with Address Autofill).
Added markers and the City-Search-Option to the map.

### CODE REVIEW

File: app/views/books/_info_window.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<h2><%= book.title %></h2>
```
Comment (syboulette) [inline app/views/books/_info_window.html.erb:1]: What is this new page about?
Who can see it?
Thanks

File: app/views/books/_marker.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="badge rounded-pill bg-primary fs-6">
```
Comment (syboulette) [inline app/views/books/_marker.html.erb:1]: Same here, what is this HTML code for? thanks :)

File: db/seeds.rb [hunk 3]
```diff
@@ -24,10 +24,11 @@
     author: Faker::Book.author, #=> "Alysha Olsen"
     genre: Faker::Book.genre, #=> "Mystery"
     date_of_publication: Faker::Date.backward(days: 100_00),
-    language: ["English", "Portuguese", "French", "Spanish", "German"].sample.to_s,
+    language: ["English", "Portuguese", "French", "Spanish", "German"].sample,
     daily_price: rand(0.01..10.00),
     user_id: User.all.sample.id,
-    description: Faker::Quotes::Shakespeare.hamlet_quote
+    description: Faker::Quotes::Shakespeare.hamlet_quote,
+    address: ["Berlin", "London", "Munich", "New Delhi", "Paris", "Miami"].sample
```
Comment (syboulette) [inline db/seeds.rb:31]: Why not use faker city instead to have a lighter code?

**Review by syboulette — COMMENTED** [review]
Hi Ariane, thank you for the great work! if you can just answer the couple of comments that would be great! thanks :)

File: app/views/books/_info_window.html.erb (hunk 1 above)
Comment (akramerin) [reply app/views/books/_info_window.html.erb:1]: it is part of the map. when you toggle a marker, you can see the Info window (in this case: The title of the book)

File: app/views/books/_marker.html.erb (hunk 2 above)
Comment (akramerin) [reply app/views/books/_marker.html.erb:1]: it is the marker on the map (to show, where a book is) - if we want to , we can customize it more, I just left it like that.

File: db/seeds.rb (hunk 3 above)
Comment (akramerin) [reply db/seeds.rb:31]: good idea, i will check.
i tried faker address, but then the map did not show any markers, because they were not real addresses.

Comment (akramerin) [reply db/seeds.rb:31]: i am not sure if faker.city does real cities - the example city does not exist. but we need existing cities, so mapbox can show them on the map.

File: app/views/books/_info_window.html.erb (hunk 1 above)
Comment (syboulette) [reply app/views/books/_info_window.html.erb:1]: Cool! thanks :)

File: app/views/books/_marker.html.erb (hunk 2 above)
Comment (syboulette) [reply app/views/books/_marker.html.erb:1]: Brill Thanks :)

File: db/seeds.rb (hunk 3 above)
Comment (syboulette) [reply db/seeds.rb:31]: Ok :)

**Review by syboulette — APPROVED** [review]

**Pull request merged**